app/templates/zazurc.json

```diff
@@ -1,6 +1,7 @@
 {
   "hotkey": "alt+space",
   "theme": "tinytacoteam/zazu-light-theme",
+  "primaryMonitor": false,
```

Comment:
maybe we even should use a `monitor` section... Than we can work from there...

```

"monitor": {
    "position": "primary|detect",
   "force": true|false
}

```

this is also better if we want to add more in the monitor section in the near future